.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 🚀 Feature
+about: Submit a proposal for a new feature
+labels: "feature"
+---
+
+## 🚀 Feature
+
+(A clear and concise description of what the feature is.)
+
+### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
+
+(Write your answer here.)
+
+## Motivation
+
+(Please outline the motivation for the proposal.)
+
+## Pitch
```

Comment:
I think motivation and pitch are two very broad terms. Would be better if you could replace them with more specific questions and also replace contributing guidelines with the code of conduct